includes/class-content-toc.php

```diff
@@ -113,7 +113,7 @@ public function shortcode( $atts, $content = null ) {
 						'<li class="hm-toc-item-%s"><a href="#heading-%d">%s</a></li>',
 						esc_attr( $item[2] ),
 						esc_attr( $key_current ),
-						esc_html( $item[1] )
+						esc_html( strip_tags( $item[1] ) )
```

Comment:
You could have a variable called `$raw_item` but I think it'd be more straight forward to just pass the stripped version of the string.

Can't think of a reason why someone would need to keep any HTML from a heading for the TOC output.